recipes/mpb/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "mpb" %}
+{% set version = "1.6.2" %}
+{% set sha256 = "aae270f9f51e7ff852fe578b8ef775b22489d4198f6a35ee0a91a60548fd1e3a" %}
+
+package:
+  name: {{ name }}
+  version: {{version}}
+
+source:
+  fn: {{ name }}-{{version}}.tar.gz
+  url: https://github.com/stevengj/{{ name }}/releases/download/v{{ version }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - openblas
+    - gcc
+    - fftw
+    - hdf5
+    - libctl >=4.0.0
+  run:
+    - openblas
```

Comment:
Also `0.2.20|0.2.20.*.` here as well.

Reply:
@jakirkham - I thought this was something we got with conda-build 3 and the conda-forge-pinnings

Reply:
I mostly agree here, but not quite.

I'd advise an alternate pattern. In the build deps, remove all pins. Conda-build 3 will do the right thing here.

In the run deps, you won't get automatic pinning until the upstreams are edited to include run_exports and rebuilt. So, for those, you should do:

- {{ pin_compatible('openblas', max_pin='x.x.x') }}
- {{ pin_compatible('hdf5', max_pin='x.x.x') }}

These can be completely deleted in run deps once the upstreams have run_exports.

Reply:
Further discussion in [this thread]( https://github.com/conda-forge/mpb-feedstock/pull/2#discussion_r184384413 ).